plugin/web/extensions/sidebar/core/components/hooks.ts

```diff
@@ -124,7 +124,7 @@ export default () => {
         let dataToAdd = data?.find(d => d.dataID === dataset.dataID);
 
         if (!dataToAdd) {
-          dataToAdd = convertToData(dataset);
+          dataToAdd = convertToData(dataset as DataCatalogItem);
```

Comment:
Can you clean up this function (handleProjectDatasetAdd) based on the UserDataItem type update.
I misunderstood and thought myData data shouldn't be shown at all in the sidebar, and only added to the scene, so the first if statement does just that.

Reply:
I fixed!
[fix: type](https://github.com/eukarya-inc/reearth-plateauview/pull/146/commits/f9c30831402b82b529e3161ec9e844fa2eae35b9)

Reply:
I fixed.
[fix: remove if statement](https://github.com/eukarya-inc/reearth-plateauview/pull/146/commits/66986cb210016d07f15283039df1ad779ac55154)